ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/dataflow_utils.py

```diff
@@ -36,24 +37,24 @@ def get_authenticated_client(client, app_name=SAMPLE_APP_NAME,
         token_key = dataflow_session.token_key
 
     token_path = _get_token_path_(app_name, token_key)
+    print("Token with key [***] is available at path [***].")
```

Comment:
Is [***] appropriate for logging at all? Maybe thinking about logging in another way?

Reply:
Yeah, this is similar to masking the inputs when user trying to add password on any login webpage. It basically specifies that the user entered some confidential detail and did not left blank but to acknowledge that something was typed, the characters are replaced with `*` which might not might not match the length of the string type. So, it is appropriate to log this way.